oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -560,7 +560,19 @@ impl Runtime {
         node_id: NodeId,
         label: &Label,
     ) -> Result<(oak_abi::Handle, oak_abi::Handle), OakStatus> {
+        info!("creating Channel with label: {:?}", label);
```

Comment:
There's already a `debug!` log in `RuntimeProxy` that includes the label information &ndash; is this needed?  (`info` level is also higher than the convention in `proxy.rs`, which logs all API calls with a pair of `debug` logs with a standard format).

Reply:
Good catch, removed for now, though I think we should remove those logs from `RuntimeProxy` and only keep them in `Runtime` itself? I'll create a separate PR for that anyways.